### PR TITLE
CMR-9392 adding Grid Draft records to the CMR.

### PR DESCRIPTION
**Pull request by eereiter**

### CODE REVIEW

File: ingest-app/src/cmr/ingest/api/generic_documents.clj [hunk 1]
```diff
@@ -56,9 +56,7 @@
   {:collection-draft :umm-c
    :service-draft :umm-s
    :tool-draft :umm-t
-   :variable-draft :umm-var
-   :data-quality-summary-draft :data-quality-summary
-   :order-option-draft :order-option})
```
Comment (sxu123) [inline ingest-app/src/cmr/ingest/api/generic_documents.clj:61]: Why are these removed? rather than adding :grid-draft?  I'm confused...

**Review by sxu123 — APPROVED** [review]

File: schemas/resources/schemas/grid-draft/v1.0.0/search.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                "native\_id": "sampleNative-Id",
+                "name": "grid-name"
+            }
+        ]
+    }
+```
+
+##### UMM JSON
+
+The UMM JSON response contains meta-metadata of the Variable Draft, the UMM fields and the associations field if applicable. [To search over specific versions of UMM](#umm-json).
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/grid-draft/v1.0.0/search.md:120]:
```suggestion
The UMM JSON response contains meta-metadata of the Grid Draft, the UMM fields and the associations field if applicable. [To search over specific versions of UMM](#umm-json). 
```

File: schemas/resources/schemas/grid-draft/v1.0.0/search.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            </reference>
+        </references>
+    </results>
+```
+
+##### JSON
+
+The JSON response includes the following fields.
+
+* hits - How many total Order Options were found.
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/grid-draft/v1.0.0/search.md:84]:
```suggestion
* hits - How many total Grids were found.
```

File: schemas/resources/schemas/grid-draft/v1.0.0/metadata.json [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            "URLContentType": "C-Type",
+            "Type": "Type"
+        }
+    ],
+    "AdditionalAttribute": {
+        "Name": "attribute-1",
+        "Description": "Sample",
+        "DataType": "STRING"
+    }
+}
```
Comment (eudoroolivares2016) [inline schemas/resources/schemas/grid-draft/v1.0.0/metadata.json:118]: Add newline

File: schemas/resources/schemas/grid-draft/v1.0.0/search.md (hunk 2 above)
Comment (eereiter) [reply schemas/resources/schemas/grid-draft/v1.0.0/search.md:120]: Fixed.

File: schemas/resources/schemas/grid-draft/v1.0.0/search.md (hunk 3 above)
Comment (eereiter) [reply schemas/resources/schemas/grid-draft/v1.0.0/search.md:84]: Fixed

File: schemas/resources/schemas/grid-draft/v1.0.0/metadata.json (hunk 4 above)
Comment (eereiter) [reply schemas/resources/schemas/grid-draft/v1.0.0/metadata.json:118]: Fixed.

File: ingest-app/src/cmr/ingest/api/generic_documents.clj [hunk 5]
```diff
@@ -56,9 +56,7 @@
   {:collection-draft :umm-c
    :service-draft :umm-s
    :tool-draft :umm-t
-   :variable-draft :umm-var
-   :data-quality-summary-draft :data-quality-summary
```
Comment (eudoroolivares2016) [inline ingest-app/src/cmr/ingest/api/generic_documents.clj:60]: I'm assuming that the mapping for the generic concept type to draft concept type can just be pulled out and is being done so somewhere else

**Review by eudoroolivares2016 — APPROVED** [review]
